test/test-fsd-test.js

```diff
@@ -1,36 +1,18 @@
-/* global describe it expect before beforeEach afterEach */
-const { home } = require('../controller/env');
-const os = require('node:os');
-const path = require('path');
+/* global describe it before afterEach */
 require('mocha');
 require('should');

```

Comment:
are these requires necessary?
```js
require('mocha');
require('should');
```

Reply:
I'm not sure. I made (or will make) an issue to update Mocha. I started it on this PR but it broke everything, so I reverted for now.

Reply:
for sure. I'm pretty sure when you run the tests with `mocha`, that mocha exists in the global scope. It may be that these are meant to be run for projects (within the tool) and not the tests for the tool.. you could probably answer that more clearly, since you've been poking around the tool